ex-sdl-cairo-freetype-harfbuzz.c

```diff
@@ -252,8 +259,9 @@ int main () {
     }
 
     FT_Done_FreeType(ft_library);
-
-    SDL_FreeSurface(sdl_surface);
+    //SDL_DestroyTexture(texture); # I don't know why I can't destroy this :S
```

Comment:
Probably a good thing to know and figure out how to do! :) Everything we allocate should be freed. It's been awhile since I've delved into this stuff so I'd have to go RTFM if there's a problem freeing it here.

Reply:
I opted to freeing and reallocating `texture` inside the loop for now.